lib/actions-oclif/env/rename.ts

```diff
@@ -70,9 +69,7 @@ export default class EnvRenameCmd extends Command {
 		},
 	];
 
-	// hardcoded 'env rename' to avoid oclif's 'env:rename' topic syntax
-	public static usage =
-		'env rename ' + new CommandHelp({ args: EnvRenameCmd.args }).defaultUsage();
+	public static usage = 'env rename <name> <value>';
```

Comment:
Was it supposed to be `<id>` instead of `<name>`?
```
$ ./bin/balena help env rename
...
USAGE
  $ balena env rename <name> <value>

ARGUMENTS
  <id>     variable's numeric database ID
  <value>  variable value; if omitted, use value from this process' environment
```
See also diff for `cli.markdown`.

Reply:
hmm, will check